src/navigation/AppNavigator.tsx

```diff
@@ -79,15 +62,8 @@ export default AppNavigator;
 const ApiLoadingStack = createStackNavigator();
 function ApiLoadingNavigator() {
   return (
-    <ApiLoadingStack.Navigator
-      headerMode={'none'}
-      mode={'modal'}
-      screenOptions={{gestureEnabled: false}}>
-      <ApiLoadingStack.Screen
-        name={'ApiLoadingPage'}
-        component={ApiLoadingScreen}
-        options={{gestureEnabled: false}}
-      />
+    <ApiLoadingStack.Navigator headerMode={'none'} mode={'modal'} screenOptions={{gestureEnabled: false}}>
+      <ApiLoadingStack.Screen name={'ApiLoadingPage'} component={ApiLoadingScreen} options={{gestureEnabled: false}} />
```

Comment:
nit: We should probably drop `"Page" `to be consistent.. I got used to call them `pages` in Flutter as well :) but in RN `"screens"` are usually the standard.

Reply:
done ✅